libevmasm/GasMeter.h

```diff
@@ -67,16 +67,29 @@ namespace GasCosts
 	}
 	static unsigned const keccak256Gas = 30;
 	static unsigned const keccak256WordGas = 6;
+	static unsigned const coldSloadCost = 2100;
+	static unsigned const coldAccountAccessCost = 2600;
+	static unsigned const warmStorageReadCost = 100;
```

Comment:
Can you rename this one to `warmSloadCost`, or the previous one to `coldStorageReadCost` so the naming is consistent?

Reply:
Ok, adding a comment if fine in that case